reactioncommerce/reaction
# chore: remove meteor app-tests

Resolves parts of #5529
Impact: **major**  
Type: **chore**

## Issue
We are in the process of removing meteor from the node-app. There are a few meteor app-test files which we are currently refactoring one-by-one into unit tests, where necessary. The `app-test` CI takes a long time to load, and there are frequent issues with these tests.

## Solution
Instead of removing these tests one by one as go through the process of demeteorization, we will remove them all here to allow for quicker dev, and no need to go back and deal with broken meteor tests that are / will soon be irrelevant. This also cuts CI running time, with the full process, including docker build and docker push, taking less than 20 minutes now: 

![chore__remove_meteor_app-tests_by_kieckhafer_·_Pull_Request__5560_·_reactioncommerce_reaction](https://user-images.githubusercontent.com/4482263/64977682-bbc59f80-d868-11e9-859c-8476b3d03cdb.png)

Locally, tests run in under 9 minutes.

<img width="422" alt="docker" src="https://user-images.githubusercontent.com/4482263/64978490-773b0380-d86a-11e9-9456-0ccb362f6063.png">

++

<img width="397" alt="bash" src="https://user-images.githubusercontent.com/4482263/64978377-38a54900-d86a-11e9-83bd-c229fa7cda4e.png">

## Breaking changes
None

## Testing
1. See that the app starts
1. See that tests run locally
1. See that CI runs on GitHub, without `test-app` running
